internal/services/compute/shared_image_resource.go

```diff
@@ -68,6 +70,25 @@ func resourceSharedImage() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"disallowed_disk_types": {
```

Comment:
could we change this to 
```suggestion
			"disk_types_not_allowed": {
```

```suggestion
			"disallowed_disk_types": {
```

Reply:
Updated